B827EBFFFEF9A421.json

```diff
@@ -0,0 +1,26 @@
+  
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEF9A421",
+    "servers": [
+      {
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      },
+      {
```

Comment:
don't connect the same gateway to both v2 and v3. The two versions are anyway interconnected.

Reply:
I'm a little clueless. I hope this is what you meant